README.md

```diff
@@ -37,3 +37,12 @@ You can use this module with the [FileVault Package Maven Plugin][3] in version
 [1]: https://jackrabbit.apache.org/jcr/node-type-notation.html
 [2]: https://jackrabbit.apache.org/filevault/validation.html
 [3]: https://jackrabbit.apache.org/filevault-package-maven-plugin/index.html
+
+# Provided Versions
+
+| Version   |  Type   |  Description |
+| --------- | ------- | ------------ |
+| 6.5.5.0   | Classic | Exported from AEM 6.5.5, has not changed for AEM 6.5.6 |
+| 2020.09.0 | Cloud   | Exported from AEM SDK 2020.9.4194.20200909T200349Z-200827 |
```

Comment:
I would maybe just use the exact same version, albeit `2020.9.4194.20200909T200349Z-20082` is long there is then no confusion about what exactly `2020.09.0` refers to

Reply:
That would require copy&paste as no one can remember such a long string. IMHO not that relevant as no one will be on exactly that version.
Why do you think it is relevant?